src/with-urql-client.tsx

```diff
@@ -1,17 +1,16 @@
+import { NextComponentClass, NextContext, NextFC } from 'next';
 import React from 'react';
-import { NextComponentClass, NextFC, NextContext } from 'next';
 import ssrPrepass from 'react-ssr-prepass';
 import {
-  Provider,
+  cacheExchange,
   Client,
   ClientOptions,
   dedupExchange,
-  cacheExchange,
-  fetchExchange,
   Exchange,
+  fetchExchange,
+  Provider,
 } from 'urql';
-import { SSRExchange, SSRData } from 'urql/dist/types/exchanges/ssr';
-
+import { SSRData, SSRExchange } from 'urql/dist/types/exchanges/ssr';
```

Comment:
Do you have some sort of plugin enabled that sorts `import`s alphabetically? I'd prefer to remove these from the diff if you're ok with that as it's not meaningful to the changes being introduced.

Reply:
upp, yeah vscode is doing that for me, will remove it